staging/cse/windows/kubeletfunc.ps1

```diff
@@ -209,6 +209,24 @@ function Get-KubePackage {
     Remove-Item $zipfile
 }
 
+function Add-KubeletNodeLabel {
+    Param(
+        [Parameter(Mandatory=$true)][string]
+        $KubeletNodeLabels,
+        [Parameter(Mandatory=$true)][string]
+        $Label
+    )
+
+    $labelList = $KubeletNodeLabels -split ","
+    foreach ($existingLabel in $labelList) {
+        if ($existingLabel -eq $Label) {
+            return $KubeletNodeLabels

```

Comment:
Add logs here and line 226 so we can know when the label is added for troubleshooting?

Reply:
added